client/src/js/LeftAndMain_Subsites.js

```diff
@@ -11,16 +11,6 @@
       }
     });
 
-    /*
-     * Reload subsites dropdown when links are processed
-     */
-    $('.cms-container .cms-menu-list li a').entwine({
```

Comment:
Not only is this selector wrong (should be `.cms-menu__list`) but this just straight up isn't necessary. Clicking on the links in the left menu reloads the whole left menu anyway.